lib/pages/event_creation/event_creation.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_database/firebase_database.dart';
+import 'package:flutter/material.dart';
+import 'package:regatta_buddy/pages/event_creation/event_form.dart';
+import 'package:regatta_buddy/pages/event_creation/event_route.dart';
+import 'package:regatta_buddy/pages/event_creation/event_social.dart';
+import 'package:regatta_buddy/pages/home.dart';
+import 'package:regatta_buddy/widgets/app_header.dart';
+import 'package:regatta_buddy/widgets/complex_marker.dart';
+import 'package:uuid/uuid.dart';
+
+class EventCreationPage extends StatefulWidget {
+  static const String route = '/event_creation';
```

Comment:
Lets stay with our current convention and use camel case
https://github.com/SomeBuddies/RegattaBuddy/blob/master/lib/pages/regatta_details.dart#L7

Reply:
My convention was to write this with underscore, but I don't believe it's important. The goal is to use PageName.route so it is does not matter what's inside